src/libs/ReportUtils.js

```diff
@@ -3592,6 +3592,46 @@ function getReportPreviewDisplayTransactions(reportPreviewAction) {
     );
 }
 
+/**
+ * Return iou report action display message
+ *
+ * @param {Object} reportAction report action
+ * @returns {String}
+ */
+function getIouReportActionDisplayMessage(reportAction) {
+    const originalMessage = _.get(reportAction, 'originalMessage', {});
+    let displayMessage;
+    if (originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY) {
```

Comment:
I don't think we do, but for safer side, do we want to check `if (_.isEmpty(originalMessage))` ?

cc - @joelbettner ?

Reply:
I think we can add the check or return early if no originalMessage is found, but it should not happen really